src/core/web_api/p2_api.js

```diff
@@ -123,17 +123,8 @@ class Auth0APIClient {
     this.getUserInfo(token, callback);
   }
 
-  getSSOData(cb) {
-    if (this.isUniversalLogin) {
```

Comment:
This logic is no longer required?

Reply:
nope, auth0.js has all the logic to handle universal login